sponsor.html

```diff
@@ -0,0 +1,156 @@
+---
+layout: page
+title: Sponsor Us
+permalink: /sponsor/
+---
+<div class="panel panel-header panel-sponsor">
+  <div class="hero-container panel-container">
+    <div class="fixed-width">
+      <div class="hero-text">
+        <h1 class="header">Help us fund the future of Student Robotics</h1>
+        {% include donate.html %}
+      </div>
+    </div>
+  </div>
+</div>
+<div class="panel">
+  <div class="panel-container">
+    <div class="fixed-width">
+      <h2 class="text-center">Become a Sponsor</h2>
+      <div class="row text-center">
+        Student Robotics is looking for new sponsors to help us fulfil our mission.
+        The support from our sponsors will make it possible to sustain our programme
+        and continue to run free to enter competitions that attract students from
+        a wide range of schools and colleges. By partnering with us, you will be
+        able to increase your organisation or product awareness with the next
+        generation of engineers.
+      </div>
+    </div>
+  </div>
+</div>
+<div class="panel panel">
+  <div class="panel-container">
+    <div class="fixed-width">
+      <h2 class="text-center">Sponsor Levels</h2>
+      <div class="row">
+        <div class="column s-4-12 sponsor-level">
+          <h3>Bronze</h3>
+          <h4>£500 or more</h4>
+          <ul>
+            <li>Link and logo on our website</li>
+            <li>Company name, logo and brief overview in our competition booklet</li>
+            <li>Invite for a representative to attend the competition</li>
+          </ul>
+        </div>
+        <div class="column s-4-12 sponsor-level">
+          <h3>Silver</h3>
+          <h4>£1,000 or more</h4>
+          <ul>
+            <li>All the perks of Bronze sponsorship</li>
+            <li>Friendly mention on social media</li>
+            <li>Logo on a banner at the competition</li>
+            <li>An opportunity during the year to present your organisation to our student competitors</li>
+          </ul>
+        </div>
+        <div class="column s-4-12 sponsor-level">
+          <h3>Gold</h3>
+          <h4>£2,000 or more</h4>
+          <ul>
+            <li>All the perks of Silver sponsorship</li>
+            <li>Logo on the volunteer t-shirts</li>
+            <li>Friendly mention and thank you at the competition</li>
+            <li>Sponsor an award - put your name to one of our existing awards, or suggest a new award and define the judging criteria*</li>
```

Comment:
Is the trailing asterisk on this line accidental, or are there some additional terms which need mentioning?

Reply:
There were additional terms in the sponsorship google drive doc that I forgot to actually copy and place somewhere on this page

Reply:
```suggestion
            <li>Sponsor an award - put your name to one of our existing awards, or suggest a new award and define the judging criteria</li>
```